src/_sass/docs.scss

```diff
@@ -21,13 +21,13 @@ $sidebar-maximum-width: 400px;
 .Docs-wrapper {
   display: flex;
   flex-direction: column;
-  margin-top: $spacing-unit;
+  margin-top: 0px;
```

Comment:
You can remove `margin-top: 0px;` entirely and have the same layout, since `.Docs-wrapper` has no margin by default.